1.10/Dockerfile.rhel7

```diff
@@ -32,7 +32,10 @@ LABEL summary="$SUMMARY" \
 ENV NGINX_CONFIGURATION_PATH=/opt/app-root/etc/nginx.d
 ENV NGINX_DEFAULT_CONF_PATH=/opt/app-root/etc/nginx.default.d
 
-RUN yum install -y yum-utils && \
+# To use subscription inside container yum command has to be run first (before yum-config-manager)
+# https://access.redhat.com/solutions/1443553
+RUN yum repolist > /dev/null && \
```

Comment:
And I can replicate the issue the same way even on a plain `rhel7` image (7.4 based). So this might be a bug in the base image, or maybe in one of the yum packages.